docs/03-tutorials/03-set-up-idp.md

```diff
@@ -0,0 +1,114 @@
+For example, if you want to use `k3d` you need to pass extra `--k3s-server-arg` flags containing oidc server configuration at the cluster creation. See [specs](https://k3d.io/usage/commands/k3d_cluster_create/) of the `k3d cluster create` command, i.e:
+
+```bash
+k3d cluster create kyma \
+--k3s-server-arg "--kube-apiserver-arg=oidc-issuer-url=<your-ipd-issuer-url>" \
+--k3s-server-arg "--kube-apiserver-arg=oidc-username-claim=<username-claim-at-your-ipd>" \
+--k3s-server-arg "--kube-apiserver-arg=oidc-client-id=<your-ipd-client-id>" \
+--k3s-server-arg "--kube-apiserver-arg=oidc-groups-claim=<group-claim-at-your-ipd>" \
+```
+
+If you use managed kubernetes you will do it different depending on your provider.
+For example, if you use gardener as a managed kubernetes offering you will probably wnt to look at the [OIDCPreset](https://github.com/gardener/gardener/blob/master/docs/usage/openidconnect-presets.md) resources that help you with the task.
+
+
+
+### Configure Role Based Access to identies provided by your OIDC server
+
+Now, whenever api server is called with a JWT token, the api server will be able to validate and extract the associated identity ( from the `username` and `group` claims of the JWT token).
+
+You shoud now define which individuals or groups should have access to which kyma resources because by default they won't have access to anything. You need to model permissions using the concept of Role Based Access Control ([RBAC](https://kubernetes.io/docs/reference/access-authn-authz/rbac/)).
+
+By default, Kyma comes with the following ClusterRoles:
+
+- **kyma-admin**: gives full admin access to the entire cluster
+- **kyma-namespace-admin**: gives full admin access except for the write access to [AddonsConfigurations](/components/helm-broker#custom-resource-addons-configuration)
+- **kyma-edit**: gives full access to all Kyma-managed resources
+- **kyma-developer**: gives full access to Kyma-managed resources and basic Kubernetes resources
+- **kyma-view**: allows viewing and listing all of the resources of the cluster
+- **kyma-essentials**: gives a set of minimal view access right to use the Kyma Console
+
+To bind a user to the **kyma-admin** ClusterRole, run this command:
+
+```
+kubectl create clusterrolebinding {BINDING_NAME} --clusterrole=kyma-admin --user={USERNAME AS IDENTIFIED AT YOUR IDP}
+```
+
+To check if the binding is created, run:
+
+```
+kubectl get clusterrolebinding {BINDING_NAME}
+```
+
+To bind a group of users to the **kyma-admin** ClusterRole, run this command:
+
+```
+kubectl create clusterrolebinding {BINDING_NAME} --clusterrole=kyma-admin --group={GROUPNAME}
+```
+
+You can combine user and group level permission in one binding. Run `kubectl create clusterrolebinding --help` in your terminal for more options.
+
+### Configure kubectl access
+
+With this step you will set up the OIDC provider in  kubeconfig file to enforce authentication flow when accessing kyma via `kubectl`.
```

Comment:
```suggestion
With this step, you will set up the OIDC provider in the kubeconfig file to enforce authentication flow when accessing Kyma using `kubectl`.
```

Reply:
Rather "using" than "via -> https://kyma-project.io/community/guidelines/content/#style-and-terminology-style-and-terminology-terminology